modules/annotations/src/main/java/org/mule/config/expression/ExpressionFilterParser.java

```diff
@@ -26,6 +27,14 @@
  */
 public class ExpressionFilterParser
 {
+
+    private MuleContext muleContext;
+
+    public ExpressionFilterParser(MuleContext muleContext)
```

Comment:
Not sure what javadoc is required "Creates a new instance of ExpressionFilterParse is implicit/obvious" especially given there is only a single constructor.